### PR TITLE
Ember Best Practice: Each/else Handlebars conditional

### PR DESCRIPTION
**Pull request by duggiefresh**

### CODE REVIEW

File: source/posts/2015-11-23-ember-practices-each-else.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title: "Ember Best Practices: Handlebars' Overlooked {{#each}} {{else}} Conditional Block"
```
Comment (michaeldupuisjr) [inline source/posts/2015-11-23-ember-practices-each-else.md]: I think this should be **Handlebars's** if Handlebars is considered singular possessive.

**Comment by michaeldupuisjr** [comment]
Minor comment, otherwise, LGTM.

File: source/posts/2015-11-23-ember-practices-each-else.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+---
+
+In the past few months, my esteemed coworkers have written about [various
+Ember best practices][best-practices]. To follow suit, I'd like to
+reintroduce an often forgotten Handlebars trick.
+
+How many times have you stumbled across a template that wrapped an
+`{{#each}}` block with an outer `{{#if}}` conditional block?
+
+The Verbose
```
Comment (danmcclain) [inline source/posts/2015-11-23-ember-practices-each-else.md]: Why not make these h2's instead `## The Verbose`

**Comment by duggiefresh** [comment]
Updated per comments, thanks @michaeldupuisjr and @danmcclain !

**Comment by bcardarella** [comment]
:+1: thank you for getting this in today

**Pull request merged**